File: SparkApplication/src/main/java/Models/Student.java
package Models;

public class Student extends EntityBase {

    private String firstName;
    private String lastName;
    private String groupId;

    public Student(){
        super();
    }

    public String getFirstName() {
        return firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public String getGroupId() {
        return groupId;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public void setGroupId(String groupId) {
        this.groupId = groupId;
    }

    @Override
    public String toString() {
        return getId()+","+getType()+","+getFirstName()+","+getLastName()+","+getGroupId();
    }
}

File: WebApi/Models/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebApi.Models
{
    public class Student:EntityBase<Student>
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public List<Mark> Marks { get; set; }
        //public string MiddleName { get; set; }
        public string GroupId { get; set; }
    }
}
